conf/map/battle/feature.conf

```diff
@@ -55,4 +55,11 @@ features: {
 	// Requires: 2014-10-22bRagexe or later
 	// Disabled by default while test version is out; enable at your own risk -- the mean dev.
 	roulette: false
+
+	// Pre-Renewal instances cooldown mode
+	// Fixes entrance exploits and changes behavior of instances cooldown with this values:
+	// 0 - Classic behaviour with exploit entrance (not recommended)
```

Comment:
i think it makes more sense to have 0 - fixed behavior. After all, this is supposed to be a bugfix, but with an option to go back to old behavior. Thats why i think it should be:
0 - Classic behavior without exploit entrance (default)
1 - Athena behavior with exploit entrance (not recommended)
2 - Renewal behavior to disable re-entrance on same instance